af/firefox-ios.xliff

```diff
@@ -681,48 +674,6 @@
         <target>Aangaande</target>
         <note>About settings section title</note>
       </trans-unit>
-      <trans-unit id="ActivityStream.ContextMenu.PinTopsite">
```

Comment:
Why is this all moved around? This was all in `Client/Localizable.strings` but now it seems to have moved to `Shared/Supporting Files/Localizable.strings`.